Installation.md

```diff
@@ -1,76 +1,139 @@
+## Quickstart: database, one user & one asset
 
-    `echo "FLASK_ENV=development" >> .env`
+### Preparing the time series database
 
-    `export FLASK_ENV=production`
-* If you need to customise settings, create `flexmeasures/<development|testing|staging|production>_config.py` and add required settings.
-  If you're unsure what you need, just continue for now and the app will tell you what it misses.
+* Make sure you have a Postgres (Version 9+) database for FlexMeasures to use. See `data/Readme.md` for instructions on this.
+* Tell `flexmeasures` about it:
 
-## Make a secret key for sessions
+    `export SQLALCHEMY_DATABASE_URI='postgresql://<user>:<password>@<host-address>[:<port>]/<db>'`
+  
+  (on Windows, use `set` instead of `export`)
+* Create the Postgres DB structure for FlexMeasures:
 
-    mkdir -p /path/to/flexmeasures/instance
-    head -c 24 /dev/urandom > /path/to/flexmeasures/instance/secret_key
+    `flexmeasures db upgrade`
 
-## Preparing the time series database
+Note that for a more permanent configuration, you can create your FlexMeasures configuration file at `~/.flexmeasures.cfg`.
+TODO: the configuration file can also be in the Flask instance directory.
 
-* Make sure you have a Postgres (Version 9+) database. See `data/Readme.md` for instructions on this.
-* Tell `flexmeasures` about it. Either you are using the default for the environment you're in (see `flexmeasures/utils/config_defaults`),
-   or you can configure your own connection string: In `flexmeasures/<development|testing|staging|production>_conf.py`,
-  set the variable `SQLALCHEMY_DATABASE_URI = 'postgresql://<user>:<password>@<host-address>[:<port>]/<db>'`
-* Run `flask db upgrade` to create the Postgres DB structure.
 
-## Preparing the job queue database
+### Configure environment
 
-To let FlexMeasures queue forecasting and scheduling jobs, install a Redis server and configure access to it within FlexMeasures' config file (see above). You can find the default settings in `flexmeasures/utils/config_defaults.py`.
+Set an environment variable to indicate in which environment you are operating (one out of development|testing|staging|production), e.g.:
 
-TODO: more detail
+   `echo "FLASK_ENV=development" >> .env`
 
-## Install an LP solver
+or:
 
-For planning balancing actions, the flexmeasures platform uses a linear program solver. Currently that is the Cbc solver. See the `FLEXMEASURES_LP_SOLVER` config setting if you want to change to a different solver.
+   `export FLASK_ENV=development`
 
-Installing Cbc can be done on Unix via:
+(on Windows, use `set` instead of `export`)
 
-    apt-get install coinor-cbc
+Note: The default is `production`, which will not work well on localhost due to SSL issues. 
 
-(also available in different popular package managers).
 
-We provide a script for installing from source (without requiring `sudo` rights) in [the CI Readme](ci/Readme.md).
+### Make a secret key for sessions and password salts
```

Comment:
I suggest to move this up. Running `flexmeasures db upgrade` throws `Error:  No secret key set.` otherwise.